.github/workflows/pkgci.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: PkgCI
+
+on:
+  workflow_dispatch:
```

Comment:
FYI you can test workflows using this `workflow_dispatch` trigger without needing to send a pull request. You can also run on any triggers you want from your fork, so long as you don't try to use self-hosted runners that only exist in this repository.

I usually do that whenever I'm iterating on workflows to avoid generating too much noise in the main repository and triggering unrelated workflows when they aren't configured with something like the "skip ci" that you tried ;). Workflows in IREE are configurable via git trailers: https://iree.dev/developers/general/contributing/#ci-behavior-manipulation, but we haven't built anything like that in this repository yet.

Reply:
skipci did work to skip all the workflows, but my problem was that the PkgCI workflow didn't show up for manual triggering : /

Reply:
Ah, I think https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/skipping-workflow-runs is all-or-nothing.